### PR TITLE
Added changelog, update docs, fix typo

### PR DESCRIPTION
**Pull request by ellennugroho**
Added a changelog on what's new, changed, or removed in new version (v2.0.0), add docs on how to update, fixed a typo in a path

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- Support for create and get charge for Cards
+- Support for eWallets
+- Support for Retail Outlets
+- Support for Recurring Payments
+- Support for Payouts
+
+### Changed
+
+- Class structure - now the implementation is not merged into one class but divided into each product class
+- Request to HTTP Client now use Guzzle Client instead of cURL
```
Comment (kevindavee) [inline CHANGELOG.md]: Does the user really need to know? What if they don't know about this?

Comment (ellennugroho) [reply CHANGELOG.md]: I think they need to know about HTTP client because the implication is in v2, it will add another dependency to guzzle library

Comment (kevindavee) [reply CHANGELOG.md]: Alright cool

**Comment by ellennugroho** [comment]
@kevindavee added `MIGRATE.md` to help users migrating

**Pull request merged**